tests/test_user.py

```diff
@@ -214,3 +252,23 @@ def test_anonymous_comment_profile_picture(self):
             URLValidator(profile_picture_url)
         except ValidationError:
             assert False, 'Bad URL for anonymous comment profile picture'
+
+
+@pytest.mark.usefixtures('set_user_client', 'set_user_client2', 'set_user_with_kaist_info',
+                         'set_boards', 'set_realname_article', 'set_realname_comment')
+class TestRealnameUser(TestCase, RequestSetting):
+    def test_realname_article_profile_picture(self):
+        r = self.http_request(self.user_with_kaist_info, 'get', f'articles/{self.realname_article.id}').data
+        profile_picture_url = r.get('created_by')['profile']['picture']
+        try:
+            URLValidator(profile_picture_url)
+        except ValidationError:
+            assert False, 'Bad URL for anonymous article profile picture'
```

Comment:
line 264는 `URLValidator` 인스턴스를 생성한 것일 뿐이므로 에러가 발생하지 않습니다.
`except`에서 `assert False`를 할 것이라면 `try except` 사용하지 않는 게 깔끔하겠습니다.
```suggestion
        URLValidator()(profile_picture_url)
```